.github/CODEOWNERS

```diff
@@ -0,0 +1,23 @@
+/LICENSE @dfds/ops-rotation
+/README.md @dfds/ops-rotation
+/ad-cloud-user-sanitycheck/** @dfds/ops-rotation
+/ad-groups/** @dfds/ops-rotation
+/aws-scripts/** @dfds/ops-rotation
+/capability-limitrange/** @dfds/ops-rotation
+/capability-logs-role/** @dfds/ops-rotation
+/capability-roles-update/** @dfds/ops-rotation
+/docker-python/** @dfds/ops-rotation
+/eks-cluster-scripts/** @dfds/ops-rotation
+/eks-sandbox-cluster/** @dfds/ops-rotation
+/eks-upgrade/** @dfds/ops-rotation
+/email-generator/** @dfds/ops-rotation
+/get-terraform-provider-versions/** @dfds/ops-rotation
+/github-branch-protection/** @dfds/ops-rotation
+/k8s-deprecated-api/** @dfds/ops-rotation
+/k8s-service-account-config-to-ssm/** @dfds/ops-rotation
+/manage-unused-replicasets/** @dfds/ops-rotation
+/microsoft-ubuntu-powershell/** @dfds/ops-rotation
+/offboarding/** @SEQUOIIA
```

Comment:
Maybe this should be both ops-rotation and Emil, just so that it comes up on our PR list (which is filtered by the PR request for the ops rotation).